slack-api-client/src/main/java/com/slack/api/methods/RequestFormBuilder.java

```diff
@@ -141,6 +141,16 @@ private RequestFormBuilder() {
         return form;
     }
 
+    public static FormBody.Builder toForm(AdminUsersSessionResetBulkRequest req) {
+        FormBody.Builder form = new FormBody.Builder();
+        if (req.getUserIds() != null) {
+            setIfNotNull("user_ids", req.getUserIds().stream().collect(joining(",")), form);
```

Comment:
This is mostly for my own education: given the `user_ids` argument is required for this API, what would happen if a user issued a SessionResetBulk request with `user_ids` equal to `null`?

Reply:
In the case, the server-side returns an error code indicating the lack of required arguments. This form request builder does not do any client-side validation here.